code/game/objects/items/binoculars.dm

```diff
@@ -200,12 +203,8 @@
 		if(MODE_RANGE_FINDER)
 			var/obj/effect/overlay/temp/laser_coordinate/LT = new (TU, laz_name, S)
 			coord = LT
-			to_chat(user, "<span class='notice'>SIMPLIFIED COORDINATES OF TARGET. LONGITUDE [coord.x]. LATITUDE [coord.y].</span>")
+			to_chat(user, "<span class='notice'>COORDINATES: LONGITUDE [coord.x]. LATITUDE [coord.y].</span>")
 			playsound(src, 'sound/effects/binoctarget.ogg', 35)
-			while(coord)
-				if(!do_after(user, 5 SECONDS, TRUE, coord, BUSY_ICON_GENERIC))
-					QDEL_NULL(coord)
-					break
```

Comment:
why was this removed?

Reply:
Served no function.

Reply:
This served to clean up the laser once you were finished with it.

This along with removing the coord check https://github.com/tgstation/TerraGov-Marine-Corps/pull/3910/files#diff-0601e714d232ec89ad6d88796b9e0ccaL151 will just allow someone to spam to flood with lasers.

I'd rather just keep the check or replace it at least with a `QDEL_NULL` instead.

Reply:
You are correct. I removed the function to prevent it from making new lasers every 5 seconds but I forget you need QDEL to delete things. Fixed now.